Comp2024/.pathplanner/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "robotWidth": 0.9,
```

Comment:
These are PathPlanner default settings. We should copy the settings for our robot into the PathPlanner settings for all of these, so the generated paths match the abilities of the robot.

Reply:
Although this can be done after this gets merged.